src/Components/LogTable.js

```diff
@@ -64,6 +65,11 @@ export class LogTable extends Component {
 			};
 
 			return (
+				<ReactCSSTransition
+					key={i}
+					classNames="highlight"
+					timeout={{ enter: 15000, exit: 1500 }}
```

Comment:
As a sanity check (I don't know much about React): you add 15000 vs 1500, and below it was 1500 and 1500. Is that intended?